cozy.yml

```diff
@@ -12,4 +12,4 @@ registries:
   default:
   - https://apps-registry.cozycloud.cc/
   - https://apps-registry.cozycloud.cc/banks
-
+  - https://apps-registry.cozycloud.cc/selfhosted
```

Comment:
I'm surprised by the order: I'd have put the selfhosted space first, and the default in last position.